src/h3lib/include/h3api.h.in

```diff
@@ -102,7 +101,7 @@ typedef enum {
     E_MEMORY = 13,  // Necessary memory allocation failed
     E_MEMORY_BOUNDS = 14,  // Bounds of provided memory were not large enough
     E_OPTION_INVALID = 15  // Mode or flags argument was not valid.
-} H3ErrorCodes;
+} H3Error;
```

Comment:
`enum` does not have an explicit size in C, rather it is implementation-defined, meaning this structure could be different sizes on different architectures. I am not seeing a particular reason to use less specific types for the return code. We document the size of the error code at https://h3geo.org/docs/next/library/errors#h3error-type.

Perhaps that also argues we shouldn't use `int` in this file either, which I have moved away from as time goes on.

Reply:
Ok, that makes sense. Its just a bummer that C doesn't give you a way to do this.

I'll close this out.